inveniordm_py/records/metadata.py

```diff
@@ -7,27 +7,34 @@
 
 """Record metadata classes."""
 
-from ..metadata import ListMetadata, Metadata
+from inveniordm_py.metadata import ListMetadata, Metadata
 
 
 class RecordMetadata(Metadata):
-    accept = "application/vnd.inveniordm.v1+json"
-    content_type = "application/vnd.inveniordm.v1+json"
+    """Record metadata class."""
+
+    accept = "application/json"
```

Comment:
Question: Why did we remove content_type for this, is it because we just use DraftMetadata for post everytime?

Reply:
It is used only when we publish a draft. However, I agree this should be explicit so I am adding it back.